.golangci.yaml

```diff
@@ -167,8 +167,8 @@ linters:
     - unparam
     - unused
     - usestdlibvars
-    - varcheck
-    #  - wastedassign
+    #  - varcheck // Fully deprecated, remove this line once an alternative is enabled for the same
+    - wastedassign
```

Comment:
This change is not in the original pr:
https://github.com/red-hat-storage/ramen/commit/b0d47022fcf762212a421a1a84224e88e61fbcd1

It sneaked in from this commit, cherry-picked later in this pr:
https://github.com/red-hat-storage/ramen/commit/1ecd9f7e835c51eebbaf8e9776ca4fdaa89760bc

I guess the second commit should come before this one.

Reply:
Reordered in latest push.